demos/motortwit/motortwit/db.py

```diff
@@ -6,7 +6,7 @@
 user = t.Dict({
     t.Key('_id'): MongoId(),
     t.Key('username'): t.String(max_length=50),
-    t.Key('email'): t.Email(),
+    t.Key('email'): t.Email,
```

Comment:
`t.Email` is not a class but already a trafaret:
https://github.com/Deepwalker/trafaret/blob/40a521ae0f580a0e5a6f3168a59cbba350d5b3ba/tests/test_keys.py#L87